src/utils/components/WorkloadProfileModal/WorkloadProfileModal.tsx

```diff
@@ -42,7 +44,7 @@ const WorkloadProfileModal: React.FC<WorkloadProfileModalProps> = React.memo(
               selections={workload}
             >
               {Object.entries(WORKLOADS_LABELS).map(([key, value]) => (
-                <SelectOption key={key} value={key}>
+                <SelectOption key={key} value={key} description={t(WORKLOADS_DESCRIPTIONS[key])}>
```

Comment:
translating a const, does not work well with i18n,
please import t as a function to the `constants.ts` file and translate the keys directly instead of translating on components, then you can also remove the translation comments above each key

Reply:
okay thanks for the info